app/assets/stylesheets/users/profile/past_donations.scss

```diff
@@ -0,0 +1,100 @@
+.profile-section[data-title="past-donations"] {
+
+  .mini-user-profile-info {
+    margin-bottom: 15px;
+    .mini-user-profile-info__pic {
```

Comment:
not a big deal, but you can save yourself some typing if you use `&__pic` when nested in scss, the `&` compile out to it's parents name (e.g. `&__pic` => `.mini-user-profile-info__pic`)

Reply:
Oh that's really nice. I had no idea. I'll definitely keep that in mind moving forward. Thanks for the tip!